<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JCOPT59" failures="3" tests="5" name="Test1" time="46.926" errors="0" timestamp="2024-09-05T22:15:16 IST">
  <testcase classname="orangehrm.dpTests.LoginTest1" name="loginTest" time="5.854"/>
  <testcase classname="orangehrm.dpTests.LoginTest1" name="loginTest" time="1.561">
    <failure type="java.lang.AssertionError" message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at orangehrm.dpTests.LoginTest1.loginTest(LoginTest1.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="orangehrm.dpTests.LoginTest1" name="loginTest" time="1.42">
    <failure type="java.lang.AssertionError" message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at orangehrm.dpTests.LoginTest1.loginTest(LoginTest1.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="orangehrm.dpTests.LoginTest1" name="loginTest" time="1.07">
    <failure type="java.lang.AssertionError" message="Login Fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Fail expected [true] but found [false]
at orangehrm.dpTests.LoginTest1.loginTest(LoginTest1.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="orangehrm.dpTests.LoginTest1" name="loginTest" time="5.836"/>
</testsuite> <!-- Test1 -->
